ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -71,6 +71,11 @@ tests:
     env:
       OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     workflow: openshift-upgrade-aws-sdn
+- as: e2e-aws-ovn-ipsec-upgrade
+  optional: true
```

Comment:
do we expect this to pass most of the time? If so, I like jobs that are required. otherwise, optional job failures don't have much affect on the dev workflow